applications/nrf_desktop/doc/nrf_profiler_sync.rst

```diff
@@ -0,0 +1,50 @@
+.. _nrf_desktop_nrf_profiler_sync:
+
+Profiler synchronization module
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Use the nrf_profiler synchronization module to synchronize the timestamps of :ref:`nrf_profiler` events between two devices connected over a physical wire.
+Timestamp synchronization is required to increase the accuracy of the measured times between the nrf_profiler events that come from two different devices.
+
+The nrf_profiler data is collected separately from both devices using dedicated Python scripts.
+The data can then be merged using the :file:`merge_data.py` script with ``sync_event`` used as a synchronization event for both ``Peripheral`` and ``Central``.
+For more details, see the :ref:`nrf_profiler` documentation.
+
+Module events
+*************
+
+.. include:: event_propagation.rst
+    :start-after: table_nrf_profiler_sync_start
+    :end-before: table_nrf_profiler_sync_end
+
+.. note::
+    |nrf_desktop_module_event_note|
+
+Configuration
+*************
+
+A predefined signal on the GPIO is used to simultaneously generate synchronization nrf_profiler events on both devices.
+For this reason, you must enable the :kconfig:option:`CONFIG_GPIO` option.
+
+You must also enable the :kconfig:option:`CONFIG_APP_EVENT_MANAGER_NRF_PROFILER_TRACER` Kconfig option.
```

Comment:
The :kconfig:option:`CONFIG_APP_EVENT_MANAGER_NRF_PROFILER_TRACER` option do not link.